incubator/redis-cache/templates/ss.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{ printf "rd-%s" .Release.Name  | trunc 20 }}"
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  serviceName: "{{ printf "rd-%s" .Release.Name  | trunc 20 }}"
+  replicas: {{default 3 .Values.replicaCount}}
```

Comment:
Format consistently, i. e. add a space after `{{` and  before `}}`.

`default 3` is redundant. It is best practice to put defaults into the `values.yaml` anyways.